oda_api/api.py

```diff
@@ -181,23 +180,30 @@
             url = "https://www.astro.unige.ch/mmoda/dispatch-data"
 
         if host is not None:
-            logger.warning(
-                "please use 'url' instead of 'host' while providing dispatcher URL")
-            logger.warning(
-                "for now, we will adopt host, but in the near future it will not be done")
+            msg = '\n'
+            msg += '----------------------------------------------------------------------------\n'
+            msg += 'support for the parameter host will end soon \n'
+            msg += 'please use "url" instead of "host" while providing dispatcher URL \n'
+            msg += '----------------------------------------------------------------------------\n'
+            warnings.warn(msg, DeprecationWarning)
             self.url = host
 
             #TODO: disregard this, but leave parameter for compatibility
             if host.startswith('http'):
                 self.url = host
             else:
-                if protocol == 'http':
-                    self.url = "http://" + host
-                elif protocol == 'https':
-                    self.url = "https://" + host
-                else:
+                if protocol != 'http' and protocol != 'https':
                     raise UserError('protocol must be either http or https')
+                else:
+                    self.url = protocol + "://" + host
         else:
+            if not oda_api.misc_helpers.validate_url(url):
+                raise UserError(f'{url} is not a valid url. \n'
```

Comment:
could you put actually usable url? try with this one, it will not work :)

Reply:
you mean: `{url} is not a valid url` -> `{url} is not a usable url` ?

Reply:
Oh no. Sorry comment went to the wrong line, I meant it for the block.
I mean example `astro.unige...` url you put in the suggestion is not a usable URL for oda api. Sorry in my comment I put dots at the end, i meant it has to be completed. It has dispatch-data at the end. It's could be just the real default url. Just try to use in oda api the url we give user as example.

Reply:
Perhaps is a good idea to have it indeed as a default url